samples/samples/testdata/singer_pb2.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
```

Comment:
Add license to all new files

Reply:
singer_pb2.py file is autogenerated by protocol buffer compiler. The similar generated files such as [spanner_pb2.py](https://github.com/googleapis/python-spanner/blob/main/benchmark/benchwrapper/proto/spanner_pb2.py), [spanner_pb2_grpc.py](https://github.com/googleapis/python-spanner/blob/main/benchmark/benchwrapper/proto/spanner_pb2_grpc.py) do not have license added in them.

Reply:
Done